source/unified-test-format/unified-test-format.rst

```diff
@@ -543,11 +545,21 @@ The structure of this object is as follows:
     ``configureFailPoint`` and any commands containing sensitive information
     (per the
     `Command Monitoring <../command-monitoring/command-monitoring.rst#security>`__
-    spec).
+    spec) unless ``observeSensitiveCommands`` is true.
 
     Test files SHOULD NOT use this option unless one or more command monitoring
     events are specified in `observeEvents <entity_client_observeEvents_>`_.
 
+  - ``observeSensitiveCommands``: Optional boolean. If true, events associated
+    with sensitive commands (per the
+    `Command Monitoring <../command-monitoring/command-monitoring.rst#security>`__
+    spec) will be observed for this client. If false or not specified, events for
+    commands containing sensitive information MUST be ignored.
+    Authentication SHOULD be disabled when this property is true, i.e.
+    `auth <runOnRequirement_auth_>`_ should be false for each ``runOnRequirement``.
+    Test runners MUST fail if ``observeSensitiveCommands`` is true and
+    authentication is enabled. See `rationale_observeSensitiveCommands`_.
```

Comment:
tl;dr: I suggest removing "Test runners MUST fail if ``observeSensitiveCommands`` is true and authentication is enabled".

I don't think this needs to be a requirement since it's adding a bit of complexity to the implementation just to protect users from occasional test failures (e.g. the example where `authenticate` was being tested for redaction and _might_ conflict with a cluster using X509 auth).

If I understand correctly, this doesn't actually care about `auth` specifically -- since it's a runtime check for what the cluster actually is. I suppose this could trigger if either `auth` was true _or_ unspecified (and the cluster happened to be using auth).

This is asking a test runner to inspect the environment in an entirely different place than when runOnRequirements might typically be resolved, so my preference would be to just get rid of this and if someone writes a bad test that unexpectedly fails we either catch that during review or after the fact when it inevitably fails on an auth environment.

I was originally going to suggest adding a "valid-fail" for this but I realize we probably can't do that since there's no guarantee the test would run against an auth-enabled cluster in the first place. Even if such a "valid-fail" test used `auth: true`, it's likely to just be skipped if the runOnRequirement(s) is not met (which would actually be bad because then we _wouldn't_ get a failure).